worlds/oot_soh/Enums.py

```diff
@@ -3453,6 +3453,8 @@ class Events(str, Enum):
     GOTTEN_LINKS_COW = "Link's Cow Was Gifted"
     CAN_ACCESS_BUGS = "Can Access Bugs"
     CAN_DEFEAT_BIG_POE = "Can Defeat Big Poe"
+    GOSSIP_STONE_FAIRY = "Gossip Stone Fairy"
+    FREE_FAIRIES = "Free Fairies"
```

Comment:
We're deviating slightly from ship here, but I've been using a single event for access to a single resource. In this case, I've suggested the name "Can Access Fairy" in another PR, and just use that across every event location granting access to a fairy that can be bottled.